articles/cosmos-db/consistency-levels.md

```diff
@@ -66,23 +66,28 @@ Strong consistency offers a linearizability guarantee. Linearizability refers to
 
 ### Bounded staleness consistency
 
-In bounded staleness consistency, the reads are guaranteed to honor the consistent-prefix guarantee. The reads might lag behind writes by at most *"K"* versions (that is, "updates") of an item or by *"T"* time interval, whichever is reached first. In other words, when you choose bounded staleness, the "staleness" can be configured in two ways:
+For multi-region, single-region write accounts, data is replicated from the primary region to all secondary (read-only) regions. For multi-region, multi-region write accounts, data is replicated from the region it was originally written in to all other writable regions. In both scenarios, while not common, there may occasionally be a replication lag from one region to another.
```

Comment:
The wording for multi-region write accounts is a bit awkward. Can we change to...

For multi-region write accounts with two or more regions, data is replicated...

Reply:
Can we also change the wording for single-region writes as well?

For single-region writes accounts with two or more regions, data is replicated...

The wording is awkward in English. I'm also concerned that if translated as is, it will be even more confusing in other languages.